docs-src/stargate-core/modules/install/partials/docker_run.adoc

```diff
@@ -38,9 +77,29 @@ docker run --name stargate \
   -e DEVELOPER_MODE=true \
   stargateio/stargate-{cass-alt-tag-40}:{stargate-docker-tag-40}
 ----
+=====
 // end::40[]
 
 // tag::dse_68[]
+.v2
+[%collapsible%open]
+===== 
+Use this docker-compose https://github.com/stargate/stargate/blob/v2.0.0/docker-compose/dse-6.8/start_cass_40_dev_mode.sh[shell script] to start the coordinator and APIs in developer mode.
+The easiest way to do that is to navigate to the `<install_location>/stargate/stargate/docker-compose` directory, and run the script.
+Currently, Stargate v2 is in beta.
+You will want to run, for example:
+
+[source,bash,subs="attributes+"]
+----
+./start_dse_{dse-alt-tag-68}_dev_mode.sh -t {stargate2-docker-tag}
+----
+
+The latest available beta image can be found https://hub.docker.com/r/stargateio/coordinator-6.8/tags[here].
```

Comment:
remove `beta`

Reply:
also should this say "found at: " instead of just "found"?

Reply:
What the text actually displays is "can be found here".

Reply:
makes sense, thanks.